### ISSUE TITLE
Use entry points instead of adding repo_path to sys.path

### ISSUE THREAD

**Issue by apayne97** [choderalab/asapdiscovery#48]
replace this:
```
repo_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
sys.path.append(repo_path)
from covid_moonshot_ml.datasets import pdb
```
with something better!

Iván has suggested this: https://packaging.python.org/en/latest/specifications/entry-points/

**Comment by ijpulidos** [comment]
This is now very much needed, I am making this a high priority one for me.

**Comment by apayne97** [comment]
Closed with #150
